pages/pipelines/branch_configuration.md.erb

```diff
@@ -2,21 +2,23 @@
 
 You can use branch patterns to ensure pipelines are only triggered when necessary. This guide shows you how to set up branch patterns for whole pipelines and individual build steps.
 
-In step-level and pipeline-level branch filtering you can use `*` as a wildcard, and `!` for not, as shown in the [examples](#branch-pattern-examples). If you want a full range of regular expressions that operate on more than branch names, take a look at the [conditionals](/docs/pipelines/conditionals) page.
+In step-level and pipeline-level branch filtering, you can use `*` as a wildcard, and `!` for not, as shown in the [examples](#branch-pattern-examples). If you want a full range of regular expressions that operate on more than branch names, take a look at the [conditionals](/docs/pipelines/conditionals) page.
 
 {:toc}
 
 ## Pipeline-level branch filtering
 
-By default a pipeline will trigger builds for all branches (`*` or blank). In your pipeline settings you can set specific branch patterns for the entire pipeline. If a commit doesn't match the branch pattern, no build will be created.
+By default, a pipeline will trigger builds for all branches (`*` or blank). In your pipeline settings, you can set specific branch patterns for the entire pipeline. If a commit doesn't match the branch pattern, no build will be created.
 
 <%= image "pipeline-level.png", width: 864, height: 306, alt: "Pipeline-level branch filtering" %>
 
-<section class="Docs__troubleshooting-note">
-  <h3>Pull Request Builds</h3>
-  <p>Builds created for pull requests will ignore any pipeline-level branch filters. If you want to limit the branches that can build pull requests, add an additional branch filter in your pipeline's source control settings.</p>
-  <p>This filter can be found under 'Build pull requests' if you have chosen the 'Trigger builds after pushing code' option.</p>
-</section>
+## Additional branch filtering for pull request builds
+
+Builds created for pull requests will ignore any pipeline-level branch filters. If you want to limit the branches that can build pull requests, add an additional branch filter in your pipeline's source control settings.
+
+This filter can be found under 'Build pull requests' if you have chosen the 'Trigger builds after pushing code' option.
```

Comment:
```suggestion
Find this filter can be found under _Build pull requests_ if you have chosen the _Trigger builds after pushing code_ option.
```

Reply:
I think these are UI elements, but if options as we discussed you might need to revert

Reply:
I'd say these are options.
![Options](https://user-images.githubusercontent.com/27306876/145617495-1e8f4a7c-4660-4681-bda8-eb55e2aad5c2.png)

Reply:
I rewrote the beginning of the sentence as suggested, but not touching the 's.